<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCharacterWindow</class>
 <widget class="QWidget" name="CreateCharacterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя персонажа</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtMale">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Мужчина</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonFemale">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Женщина</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxClass">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Офицер Службы Безопасности</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Медик</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Инженер</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Механик</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Помошник</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonHelpClass">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxStrenght">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>30</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxAgility">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>60</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxIntelligence">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>90</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelStrength">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>30</y>
     <width>49</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Сила</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAgility">
   <property name="geometry">
    <rect>
     <x>638</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ловкость</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIntelligence">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Интелект</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCreate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtoSave">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWeaponSkillName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Оружие</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMedicineSkillName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Медицина</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMechanicSkillName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Механика</string>
   </property>
  </widget>
  <widget class="QLabel" name="labClass">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>61</y>
     <width>49</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Класс</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeSskils">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Навыки</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelElectronicsSkillName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Электроника</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeTraitsName">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Особенности</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPointsLeft">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>120</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Осталось очков 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMedicineSkill">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMechanicSkill">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelElectronicsSkill">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWeaponSkill">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTraits1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>121</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTraits2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTraits3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTraitsHelp1">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTraitsHelp2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTraitsHelp3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
